includes/Core/Authentication/Authentication.php

```diff
@@ -602,16 +604,8 @@ private function handle_oauth() {
 	 * @return array Filtered $data.
 	 */
 	private function inline_js_base_data( $data ) {
-		$first_admin_id  = (int) $this->first_admin->get();
-		$current_user_id = get_current_user_id();
-
-		// If no first admin is stored yet and the current user is one, consider them the first.
-		if ( ! $first_admin_id && current_user_can( Permissions::MANAGE_OPTIONS ) ) {
-			$first_admin_id = $current_user_id;
-		}
-		$data['isFirstAdmin'] = ( $current_user_id === $first_admin_id );
-		$data['splashURL']    = esc_url_raw( $this->context->admin_url( 'splash' ) );
-
+		$data['isOwner']             = $this->owner_id->get() === get_current_user_id();
```

Comment:
See above: Here we should for now re-include `isFirstAdmin`, solely to keep the tracking logic behave like it did before. The ACs may have been somewhat unprecise here - the primary goal was to avoid actually _setting_ the owner here (persistently), which is happening in the similar logic below for `_googlesitekitLegacyData`.
```suggestion
		$data['isOwner']             = $this->owner_id->get() === get_current_user_id();
		$data['isFirstAdmin']        = $data['isOwner'] || ! $this->owner_id->get() && current_user_can( Permissions::MANAGE_OPTIONS );
```

Reply:
Updated.